src/expect.ts

```diff
@@ -16,7 +16,7 @@
 
 import type { Expect } from './expectType';
 import expectLibrary from 'expect';
-import { config, currentTestInfo } from '.';
+import { config, currentTestInfo } from './fixtures';
```

Comment:
Any reason behind this? I wanted to depend on public to make sure plugging in expect was possible externally.

Reply:
Depending on public unfortunately makes a dependency cycle: expect -> index -> spec -> expect. We can probably shuffle code in a follow up to make things nicer.